hive/db/schema.py

```diff
@@ -230,19 +231,26 @@ def setup(connection_url=_url):
     metadata.create_all(engine)
 
     conn = engine.connect()
+
     # Insert hive_blocks data
-    insert = hive_blocks.insert().values(num=0, hash='0000000000000000000000000000000000000000', prev=None, created_at='1970-01-01T00:00:00')
-    conn.execute(insert)
+    try:
+        insert = hive_blocks.insert().values(num=0, hash='0000000000000000000000000000000000000000', prev=None, created_at='1970-01-01T00:00:00')
+        conn.execute(insert)
 
-    # Insert hive_accounts data
-    insert = hive_accounts.insert()
-    conn.execute(insert, [
-        {'name': 'miners', 'created_at': '1970-01-01T00:00:00'},
-        {'name': 'null', 'created_at': '1970-01-01T00:00:00'},
-        {'name': 'temp', 'created_at': '1970-01-01T00:00:00'},
-        {'name': 'initminer', 'created_at': '1970-01-01T00:00:00'}
-    ])
+    except IntegrityError:
```

Comment:
I think it happens if the db already exists? It was always block 000000000000 that was giving the error